qiita_db/processing_job.py

```diff
@@ -931,6 +907,27 @@ def children(self):
             for jid in qdb.sql_connection.TRN.execute_fetchflatten():
                 yield ProcessingJob(jid)
 
+    @property
+    def get_validator_jobs(self):
```

Comment:
The pythonic thing would be to name this property `validator_jobs`.

Reply:
K, will change.